### PR TITLE
Feat: Add BGP dashboard

### PR DESCRIPTION
**Pull request by Mesverrum**
# Summary

New dashboard for network BGP routing neighbors using our ktranslate snmp integration 

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you check your dashboard image quality? -  Do they look good?
- [x] Did you check that your alerts actually work?
- [] Did you include an InstallPlan and Documentation reference?
- [x] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [x] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots
bgpNeighbor

### CODE REVIEW

File: dashboards/bgpNeighbor/bgpNeighbor.json [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+            "id": "viz.table"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Metric SELECT uniqueCount(Index) as Neighbors\n, uniqueCount(remote_as)\n, filter(uniqueCount(Index),where bgpPeerAdminStatus = 'start') as AdminEnabledNeighbors  \n, filter(uniqueCount(Index),where bgpPeerState = 'established') as EstablishedNeighbors\n, filter(uniqueCount(Index),where bgpPeerState != 'established' and bgpPeerState is not null and bgpPeerAdminStatus = 'start'  ) as NotEstablishedNeighbors\nfacet device_name, instrumentation.name \nwhere `mib-name` = 'BGP4-MIB' and tags.container_service = 'snmp'\n limit max"
```
Comment (zstix) [inline dashboards/bgpNeighbor/bgpNeighbor.json:27]: Just want to double-check, will this value be the same for _all_ users who install this dashboard?

```
`mib-name` = 'BGP4-MIB'
```

Comment (zstix) [reply dashboards/bgpNeighbor/bgpNeighbor.json:27]: Otherwise, this looks great! Nice job!

Comment (Mesverrum) [reply dashboards/bgpNeighbor/bgpNeighbor.json:27]: Yep, mib-name will be consistent to anyone who has this set of data.

**Comment by Mesverrum** [comment]
I was a bit confused, for cases where we just want to add the dashboard are we able to run without referencing an install plan or is that always required?  I feel like I had seen quickstarts that did and did not have them.

**Comment by aswanson-nr** [comment]
Hey @Mesverrum, you'll need an install plan to make it installable. Looks like you have that all set up!

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**